pkg/agent/openflow/pipeline.go

```diff
@@ -873,6 +892,55 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+func (c *client) allowRulesMetricFlows(conjunctionID uint32, ingress bool) []binding.Flow {
+	metricTableID := IngressMetricTable
+	offset := 0
+	// We use the 0..31 bits of the ct_label to store the ingress rule ID and use the 32..63 bits to store the
+	// egress rule ID.
+	labelRange := binding.Range{0, 31}
+	if !ingress {
+		metricTableID = EgressMetricTable
+		offset = 32
+		labelRange = binding.Range{32, 63}
+	}
+	// TODO: update in ovs-pipeline.md
+	// These two flows track the number of sessions in addition to the packet and byte counts.
+	// The flow with matching ct_state=+new calculates numbers of the sessions, bytes/packets of the first packet of a
+	// session.
+	// The flow with matching ct_state=-new calculates numbers of bytes/packets of an established connections.
```

Comment:
The flow matching 'ct_state=-new' tracks the byte/packet count of an established connection (both directions).

Reply:
Done.